### PR TITLE
Attempt fixing neverending failures on cron

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/81639

This PR:
* uses the generic logging methods in https://github.com/moderntribe/tribe-common/pull/493
* modifies the way we fetch children records to limit them to the last 4 hours

See ticket notes for more clarity.

### CODE REVIEW

File: src/Tribe/Aggregator/Records.php [hunk 1]
```diff
@@ -464,6 +469,19 @@ public function query( $args = array() ) {
 			'orderby'     => 'modified',
 			'order'       => 'DESC',
 		);
+
+		$args = (array) $args;
+
+		if ( isset( $args['before'] ) ) {
```
Comment (borkweb) [inline src/Tribe/Aggregator/Records.php]: Is `$args['before']` being used anywhere?

Comment (lucatume) [inline src/Tribe/Aggregator/Records.php]: I had not pushed all the code...

**Comment by lucatume** [comment]
waiting for https://github.com/moderntribe/tribe-common/pull/493

**Comment by barryhughes** [comment]
:bulb: remember to add changelog entries!

**Pull request merged**